full-service/src/db/models.rs

```diff
@@ -37,23 +37,40 @@ pub const TXO_INPUT: &str = "input";
 pub const TXO_OUTPUT: &str = "output";
 pub const TXO_CHANGE: &str = "change";
 
+/// An Account entity.
+///
+/// Contains the account private keys, subaddress configuration, and ...
 #[derive(Clone, Serialize, Identifiable, Queryable, PartialEq, Debug)]
 #[primary_key(id)]
 pub struct Account {
+    /// Primary key
     pub id: i32,
+    /// ???
     pub account_id_hex: String,
+    /// Private keys for viewing and spending the MobileCoin belonging to an
+    /// account.
     pub account_key: Vec<u8>,
+    /// Default subadress that is given out to refer to this account.
     pub main_subaddress_index: i64,
+    /// Subaddress used to return transaction "change" to self.
     pub change_subaddress_index: i64,
+    /// The next unused subaddress index. (Assumes indices are used sequentially
+    /// from 0).
     pub next_subaddress_index: i64,
+    /// Index of the first block where this account may have held funds.
     pub first_block: i64,
+    /// TODO: ???
```

Comment:
Not sure what this is...

Reply:
This is used by the account sync to pick up scanning at the next un-scanned block.